### DISCUSSION TITLE
How can I turn off the reminder about downloading recovery codes or adding passkey?

### DISCUSSION THREAD

**Discussion by pjhades** [community/community#78692] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello. I keep seeing the reminder on the page saying:

> Don't get locked out of your account. [Download your recovery codes](https://github.com/settings/auth/recovery-codes?two_factor_holiday_warning=true) or [add a passkey](https://github.com/settings/security?two_factor_holiday_warning=true#passkeys) so you don't lose access when you get a new device.

It shows up every time the page is reloaded. I haven't set any passkey but have downloaded my recovery codes. How can I turn off the reminder? It feels annoying. Thanks.

**Comment by DNin01** [comment]
Is this a banner notification?

Maybe if you try re-downloading your recovery codes it'll go away? Never hurts to re-download them.

**Reply by marcelolx** [reply]
That doesn't work, I already had my recovery codes downloaded and then downloaded them again, and I am still seeing the banner...

**Comment by TonyCrane** [comment]
Same problem here, it's really annoying, but I also don't why. It always appears even I set a passkey and download my recovery code. Once I close it, it will appear immediately when I refresh the page.

**Comment by dylanscott** [comment]
I'm also having the same problem, though interestingly it doesn't seem to happen in Chrome (edit: or Safari), only in Arc. It doesn't seem to be caused by extensions though - it still happens if I disable all of them.

**Comment by pjhades** [comment]
Anyway, the banner seems gone now, not sure why. I'm closing this.
